app/models/solidus_subscriptions/checkout.rb

```diff
@@ -73,7 +73,10 @@ def checkout
       apply_promotions
 
       order.checkout_steps[0...-1].each do
-        order.ship_address = ship_address if order.state == "address"
+        if order.state == "address"
```

Comment:
the line above this strikes me as...odd, is it supposed to not include the last checkout_step, because that is what [0...-1] would do 🤔

Reply:
that's correct, and that matches how checkout is for non-subscription checkouts in solidus.  it's skipping the confirm step since there's no new info added on that step

Reply:
ohhh gotcha now that makes sense